src/pudl/output/pudltabl.py

```diff
@@ -403,19 +351,10 @@ def gen_eia923(self: Self, update: bool = False) -> pd.DataFrame:
             table_name = self._agg_table_name(
                 "out_eia923__AGG_generation_fuel_by_generator"
             )
-            gen_df = self._get_table_from_db(table_name)
+            gen_df = self.get_table(table_name)
             resource = Resource.from_id(table_name)
             gen_df = gen_df.loc[:, resource.get_field_names()]
         else:
             table_name = self._agg_table_name("out_eia923__AGG_generation")
-            gen_df = self._get_table_from_db(table_name)
+            gen_df = self.get_table(table_name)
         return gen_df
-
-    ###########################################################################
-    # GLUE OUTPUTS
-    ###########################################################################
-    def epacamd_eia(self: Self) -> pd.DataFrame:
-        """Read the EPACAMD-EIA Crosswalk from the PUDL DB."""
-        return pd.read_sql("core_epa__assn_eia_epacamd", self.pudl_engine).pipe(
-            apply_pudl_dtypes, group="glue"
-        )
```

Comment:
Not sure why this was implemented as a special case, since it seems to be covered by the normal output methods. I rolled it up into the method registration above since it was the last reference to `pudl_engine`.